### ISSUE TITLE
Address already in use error with MongoDB 4 on CircleCI

### ISSUE THREAD

**Issue by iansu** [vkarpov15/run-rs#4]
I'm running this on CircleCI and I get this error:

```
[initandlisten] exception in initAndListen std::exception: listen: Address already in use, terminating
```

It looks like this is a know issue that was fixed in Mongo 3.6 and 3.7: https://jira.mongodb.org/browse/SERVER-32631

Changing `bind_ip` from `localhost` to `127.0.0.1` here seems to fix it: https://github.com/vkarpov15/run-rs/blob/master/index.js#L59

**Comment by vkarpov15** [comment]
Fixed and released v0.1.5 :+1:

**Comment by iansu** [comment]
I can confirm that fixes the problem. Thanks!
